core/src/main/java/com/scalar/db/storage/jdbc/query/SimpleSelectQuery.java

```diff
@@ -89,6 +103,51 @@ private String conditionSqlString() {
     return String.join(" AND ", conditions);
   }
 
+  private String relationalConditionSqlString() {
+    if (conjunctions.isEmpty()) {
+      return "";
+    }
+
+    List<String> conjunctionList = new ArrayList<>();
+    conjunctions.forEach(
+        conjunction -> {
+          List<String> conditions = new ArrayList<>();
+          conjunction
+              .getConditions()
+              .forEach(
+                  condition ->
+                      conditions.add(
+                          rdbEngine.enclose(condition.getColumn().getName())
+                              + convert(condition.getOperator())));
+          conjunctionList.add(String.join(" AND ", conditions));
+        });
+
+    return " WHERE " + String.join(" OR ", conjunctionList);
+  }
+
+  private String convert(ConditionalExpression.Operator operator) {
+    switch (operator) {
+      case EQ:
+        return "=?";
+      case NE:
+        return "!=?";
+      case GT:
+        return ">?";
+      case GTE:
+        return ">=?";
+      case LT:
+        return "<?";
+      case LTE:
+        return "<=?";
+      case IS_NULL:
+        return " IS NULL";
+      case IS_NOT_NULL:
+        return " IS NOT NULL";
+      default:
+        throw new IllegalArgumentException("unknown operator");
```

Comment:
How about showing the unexpected operator in the error message? I don't think it would be useful for now but for the future when we add another operator.

Reply:
Fixed in 1490ed6.